website/loaders/markdown-id-headers.js

```diff
@@ -4,21 +4,33 @@
 var slugify = require('../js/slugify');
 
 module.exports = function(md) {
-	var originalHeadingOpen = md.renderer.rules.heading_open;
-	md.renderer.rules.heading_open = function (tokens, idx, something, somethingelse, self) {
-		tokens[idx].attrs = tokens[idx].attrs || [];
-
-		var title = tokens[idx + 1].children.reduce(function (acc, t) {
-			return acc + t.content;
-		}, '');
-
-		var slug = slugify(title);
-		tokens[idx].attrs.push(['id', slug]);
-
-		if (originalHeadingOpen) {
-			return originalHeadingOpen.apply(this, arguments);
-		} else {
-			return self.renderToken.apply(self, arguments);
-		}
-	};
+    var originalHeadingOpen = md.renderer.rules.heading_open;
+    var explicitIDRegex = /[ ]?\(\(([^)]+)\)\)/;
```

Comment:
could you provide a comment for this regex :)

Reply:
Good call, added. I also removed the part where it matches the optional space at the start; spaces aren't visible, so it's not necessary to remove it.